pkg/controller/servicebindingrequest/nested/path.go

```diff
@@ -0,0 +1,110 @@
+package nested
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Field is the representation of a node in a path.
```

Comment:
can you provide more details what do you mean here by a path and a node.

Reply:
Updated.